Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM python:3.12-alpine
+LABEL authors="user"
+
+ENV PYTHONUNBUFFERED=1
+
+WORKDIR app
+
+COPY requirements.txt requirements.txt
+RUN pip install -r requirements.txt
+
+COPY . .
```

Comment:
The `COPY . .` command copies everything from the current directory to the working directory in the Docker image. Ensure that your `.dockerignore` file is correctly set up to exclude unnecessary files and directories from being copied into the Docker image.